webview/src/plots/components/templatePlots/TemplatePlotsGrid.tsx

```diff
@@ -48,8 +48,9 @@ export const TemplatePlotsGrid: React.FC<TemplatePlotsGridProps> = ({
 
   const addDisabled = useCallback(
     (e: Event) => {
-      const disabledId = (e.currentTarget as HTMLFormElement).parentElement
-        ?.parentElement?.parentElement?.id
+      const disabledId = (e.currentTarget as HTMLFormElement).closest(
```

Comment:
You would need to go back and check around https://github.com/iterative/vscode-dvc/pull/2747

Reply:
Thanks for the info! Not seeing any comments on this particular line in the prs relating to this code, but my guess on why we were using `parentElement` is because it's technically faster since `closest` keeps checking up to the `root`. I don't think this code gets fired too often (only when user's mouse enters `.vega-actions`) so should be alright to use 🤔